### PR TITLE
Add zingchart-angularjs@1.2.0 w/ npm auto-update

### LINKED ISSUE #9797: [Request] Add ZingChart-AngularJS
**Library name:** ZingChart-AngularJS 
**Git repository url:** https://github.com/zingchart/ZingChart-AngularJS
**npm package url(optional):** https://www.npmjs.com/package/zingchart-angularjs
**License(s):** MIT
**Official homepage:**
**Wanna say something? Leave message here:** cc #6754

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by NiniHsu**
close #9797, cc #9414, cc @zingchart

Pull request for issue: #9797
Related issue(s): #9414 
@kennynaoh please help me review this PR, thank you!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/zingchart/ZingChart-AngularJS
 * Official website (optional, not the repository): https://www.zingchart.com/
 * NPM package url (optional): https://www.npmjs.com/package/zingchart-angularjs
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 74
   - Count of watchers: 18
   - Count of forks: 14
 * NPM download stats (optional):
   - Downloads in the last day: 25
   - Downloads in the last week: 108
   - Downloads in the last month: 520

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/zingchart-angularjs/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "url": "https://github.com/zingchart/ZingChart-AngularJS"
+  },
+  "author": "ZingChart",
+  "license": "MIT",
+  "npmName": "zingchart-angularjs",
+  "npmFileMap": [
+    {
+      "basePath": "src",
+      "files": [
+        "**/*"
```
Comment (kennynaoh) [inline ajax/libs/zingchart-angularjs/package.json]: @NiniHsu I think it's better to use `zingchart-angularjs*` here, `**/*` basically is for `dist/` or `build/` directory.

**Comment by NiniHsu** [comment]
@kennynaoh I modified them.
Please review this PR, thanks!

File: ajax/libs/zingchart-angularjs/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "version": "1.2.0",
+  "description": "A ZingChart AngularJS directive",
+  "keywords": [
+    "zingchart-angularjs"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zingchart/ZingChart-AngularJS"
+  },
+  "homepage": "https://www.zingchart.com",
```
Comment (kennynaoh) [inline ajax/libs/zingchart-angularjs/package.json]: I think this https://zingchart.github.io/ZingChart-AngularJS/ is much better.

**Comment by NiniHsu** [comment]
@kennynaoh Please review this PR, thanks!

File: ajax/libs/zingchart-angularjs/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "zingchart-angularjs",
+  "filename": "zingchart-angularjs.min.js",
+  "version": "1.2.0",
+  "description": "A ZingChart AngularJS directive",
+  "keywords": [
```
Comment (kennynaoh) [inline ajax/libs/zingchart-angularjs/package.json]: @NiniHsu You can use the keywords here https://github.com/zingchart/ZingChart-AngularJS/blob/master/bower.json#L6

**Comment by NiniHsu** [comment]
@kennynaoh I added the keywords.
Please review the PR. Thank you!

**Comment by NiniHsu** [comment]
@kennynaoh I added the map file.
Please review the PR. Thank you!

**Comment by kennynaoh** [comment]
LGTM, ping @cdnjs/intern2

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
> close #9797, cc #9414, cc @zingchart

zingchart is an organization ... not the author.

I'll merge this one this time, but please be more careful next time.

**Pull request merged**